www/front_src/src/Header/RessourceStatusCounter/index.tsx

```diff
@@ -14,18 +14,53 @@ export const useStyles = makeStyles((theme) => ({
   link: {
     textDecoration: 'none',
   },
+  subMenuCounters: {
+    display: 'flex',
+    gap: theme.spacing(0.5),
+    [theme.breakpoints.down(768)]: {
+      display: 'grid',
+      gridTemplateColumns: 'auto auto',
+    },
+  },
+  subMenuCounters_marginLeft: {
+    marginLeft: theme.spacing(-1),
+    [theme.breakpoints.down(768)]: {
+      marginLeft: 0,
+    },
+  },
+  subMenuRight: {
+    alignItem: 'flex-start',
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.spacing(0.8),
+    justifyContent: 'space-between',
+    [theme.breakpoints.down(768)]: {
+      alignItems: 'center',
+      flexDirection: 'row',
+      gap: theme.spacing(0.1),
+    },
+  },
   subMenuToggle: {
     backgroundColor: theme.palette.common.black,
     boxSizing: 'border-box',
     display: 'none',
     left: theme.spacing(0),
-    padding: theme.spacing(1),
+    padding: '8px 8px 0 8px',
     position: 'absolute',
     textAlign: 'left',
     top: '100%',
-    width: '100%',
+    width: '135%',
+    [theme.breakpoints.down(768)]: {
+      width: '170%',
+    },
     zIndex: theme.zIndex.mobileStepper,
   },
+  subMenuToggle_extraWidth: {
+    width: '170%',
```

Comment:
Why 170%?

Reply:
From a specific breakpoint, the submenu content starts to overflow. So I have to increase the menu width to keep each "li" shown clearly in one line.